Lombiq.NodeJs.Extensions/Docs/Styles.md

```diff
@@ -60,26 +61,33 @@ Now, you can add any or all of the following entries to the `scripts` property i
+- _.prettierrc.js_: In this file you can override the above Lombiq rules, or define your own [Prettier configuration](https://prettier.io/docs/en/configuration.html) altogether.
 
-The _.stylelintrc.js_ file will automatically be created in your project during the first build. Please open it and adjust the path to _.stylelintrc.lombiq-base.js_ according to your solution's directory structure.
+The _.stylelintrc.js_ and _prettierrc.js_ files will automatically be created in your project during the first build. Please open it and adjust the path to _.stylelintrc.lombiq-base.js_ and _.prettierrc.lombiq-base.js_ according to your solution's directory structure.
+
+### Initiate Prettier auto-fix
+
+You can initiate an auto-fix for your SCSS files by running `build:styles` in the project's root directory. This will fix all Prettier violations that can be fixed automatically.
 
 ### Using a solution-wide configuration
 
 > ℹ This option only works when using Node.js Extensions from a submodule, **not** from the NuGet package.
 
-In order to use a global _.stylelintrc.js_ file for your whole solution, you can instruct Node.js Extensions to create that file in the location specified by the MSBuild property `<NodeJsExtensionsGlobalStylelintConfigurationDirectory>`. This property is easiest to add in a _Directory.Build.props_ file in your solution's root directory as follows:
+In order to use a global _.stylelintrc.js_ and _prettierrc.js_ files for your whole solution, you can instruct Node.js Extensions to create those files in the location specified by the MSBuild property `<NodeJsExtensionsGlobalStylelintConfigurationDirectory>` and `<NodeJsExtensionsGlobalPrettierStyleConfigurationDirectory>`. The easiest is to add these in a _Directory.Build.props_ file in your solution's root directory as follows:
 
 ```xml
 <NodeJsExtensionsGlobalStylelintConfigurationDirectory>$(MSBuildThisFileDirectory)</NodeJsExtensionsGlobalStylelintConfigurationDirectory>
+<NodeJsExtensionsGlobalPrettierStyleConfigurationDirectory>$(MSBuildThisFileDirectory)</NodeJsExtensionsGlobalPrettierStyleConfigurationDirectory>
 ```
 
-Details on rules can be found in the [Stylelint documentation](https://stylelint.io/user-guide/rules/list/). If you want to find out what the currently applied configuration is, coming from all the various extended configuration files, then run `npx stylelint --print-config . > rules.json` at the given location.
+Details on rules can be found in the [Stylelint documentation](https://stylelint.io/user-guide/rules/list/) and in the [Prettier documentation](https://prettier.io/docs/en/options). If you want to find out what the currently applied configuration is for Stylelint, coming from all the various extended configuration files, then run `npx stylelint --print-config . > rules.json` at the given location.
```

Comment:
> out what the currently applied configuration is for Stylelint, coming from all the various extended configuration files, then run `npx stylelint --print-config . > rules.json` at the given location.

Is there a similar thing for `Prettier`?

Reply:
Currently there isn't. https://github.com/prettier/prettier/issues/15156